flake.nix

```diff
@@ -25,6 +25,8 @@
 
           shellHook = ''
             echo "$welcomeMessage"
+            # TODO use pkgs.lib.getExe once nixpkgs are updated
+            export PROTOC_PATH=${pkgs.protobuf3_21}/bin/protoc
```

Comment:
**thought:** can't use this rn because the definition of `protobuf3_21` in the pinned version of nixpkgs has incorrect metadata and it causes the path to be derived incorrectly. This has been already fixed in the newer version of nixpkgs.

Reply:
I think we should bump to whatever major Node is being used by GHA

Reply:
It is not about GHE itself but rather the image smithy4s uses for the runner. In this case that is the latest LTS ubuntu which at the moment is 22.04. It uses nodejs 18.x as documented [here](https://github.com/actions/runner-images/blob/ubuntu22/20240526.1/images/ubuntu/Ubuntu2204-Readme.md#language-and-runtime)